src/test/java/software/amazon/encryption/s3/internal/ConvertSDKRequestsTest.java

```diff
@@ -15,8 +15,8 @@ class ConvertSDKRequestsTest {
   void testConvertPutObjectRequest_Bucket() {
     final String value = "test-bucket";
     PutObjectRequest originalRequest = PutObjectRequest.builder()
-      .bucket(value)
-      .build();
+            .bucket(value)
+            .build();
```

Comment:
nit: Indentation; it looks like this file was using 2 spaces per indentation but you've changed it to 6 here. Was that intentional?